data/planet-sources.yml

```
@@ -7,3 +7,5 @@ sources:
     url: https://blog.janestreet.com/feed.xml
   - name: hannes
     url: https://hannes.robur.coop/atom
+  - name: camlweeklynews
```

We can use the abbreviation:
```suggestion
  - name: cwn
```